.evergreen/run-mongodb-aws-test.sh

```diff
@@ -15,6 +15,9 @@ echo "Running MONGODB-AWS authentication tests"
 # ensure no secrets are printed in log files
 set +x
 
+# Write an empty prepare_mongodb_aws so no auth environment variables are set.
+echo "" > "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"
+
```

Comment:
The `run-mongodb-aws-test.sh` script is run for all scenarios for fetching AWS credentials:

```
      - func: run-aws-auth-test-with-regular-aws-credentials
      - func: run-aws-auth-test-with-assume-role-credentials
      - func: run-aws-auth-test-with-aws-credentials-as-environment-variables
      - func: run-aws-auth-test-with-aws-credentials-and-session-token-as-environment-variables
      - func: run-aws-auth-test-with-aws-EC2-credentials
      - func: run-aws-ECS-auth-test
```

This ticket only refers to EC2. I think the expected change is to only clear the `prepare_mongodb_aws.sh` script in the `run-aws-auth-test-with-aws-EC2-credentials` function.

Reply:
I see, thank you. Here are the AWS builds: https://evergreen.mongodb.com/version/6339ad28c9ec443c49aa6739